tests/Vagrantfile

```diff
@@ -0,0 +1,14 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "centos65"
+  config.vm.box_url = "http://downloads.juwai.io/devops/boxes/centos65-x86-64.box"
```

Comment:
How about just use this one https://atlas.hashicorp.com/nrel/boxes/CentOS-6.5-x86_64/versions/1.2.0 ?